util/info.c

```diff
@@ -129,15 +132,23 @@ static int str2cap(char *inputstr, uint64_t *value)
 	ORCASE(FI_TRIGGER);
 	ORCASE(FI_FENCE);
 
+	ORCASE(FI_COMPLETION);
```

Comment:
This is not a cap.  In fact, neither are the ones below this: FI_EVENT, FI_INJECT, FI_INJECT_COMPLETE, FI_TRANSMIT_COMPLETE, or FI_DELIVERY_COMPLETE.  Those need to be removed.